lib/vsc/administration/slurm/sync.py

```diff
@@ -380,16 +380,16 @@ def slurm_user_accounts(vo_members, active_accounts, slurm_user_info, clusters,
             # these are users not yet in the Slurm DB for this cluster
             new_users |= {
                 (user, vo.vsc_id, vo.institute['name'])
-                for user in (members & active_accounts) - cluster_users
+                for user in members - cluster_users
             }
 
             # these are the current Slurm users per Account, i.e., the VO currently being processed
             slurm_acct_users = {user for (user, acct) in cluster_users_acct if acct == vo_id}
 
             # these are the users that should no longer be in this account, but should not be removed
-            # we need to look up their new VO
+            # we need to look up their new VO. We remove the users which are nog longer active too.
             # Again, basic set arithmetic. LHS is the intersection of the people we have left and the active users
-            changed_users_vo = (slurm_acct_users - members) & active_accounts
+            changed_users_vo = slurm_acct_users - members - remove_users_cluster
```

Comment:
for clarity, you might want to rename `remove_users_cluster` to `inactive_users_cluster`, and add a comment that explains that inactive users are all users that exist in slurm but are not an active member in the VO (and we only retrieve active members now)